CHANGELOG_PENDING.md

```diff
@@ -7,6 +7,7 @@ Special thanks to external contributors on this release:
 ### BREAKING CHANGES:
 
 * CLI/RPC/Config
+  - [rpc] \#3113 rename n_txs to count in `/num_unconfirmed_txs` and `/unconfirmed_txs`
```

Comment:
If we're planning to release v0.29.2 and we want to include this I'd say we should make this non-breaking for now and we can fix the naming in both the rpc and the config in v0.30

Reply:
Why not release v0.30?

Reply:
To establish a better cadence of making bug fixes before making breaking changes. We should get in a habit of releasing everything on develop as non-breaking release before merging the breaking stuff

Reply:
Fair enough